packages/clerk-js/src/ui/components/SubscriptionDetails/__tests__/SubscriptionDetails.test.tsx

```
@@ -491,15 +556,29 @@ describe('SubscriptionDetails', () => {
     });
   });
 
-  it('allows cancelling a subscription of a monthly plan', async () => {
+  it.only('allows cancelling a subscription of a monthly plan', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)